### ISSUE TITLE
switch to PyQt5 and drop Python 2 support

### ISSUE THREAD

**Issue by thorade** [UdK-VPT/CoTeTo#2]
New releases of WinPython and Anaconda ship with PyQt5,
so we should probably shift from PyQt4 to PyQt5 at some point, too.
It seems it is not possible to install both in parallel.

**Comment by thorade** [comment]
It seems this would be easier if we drop Python 2 support.
For me, supporting only Python 3.5 and newer is OK.
http://www.python3statement.org/
https://docs.python.org/devguide/#status-of-python-branches

**Comment by jraedler** [comment]
A new development branch devel_v1 was opened. A lot of changes will happen here, but it will very soon break compatibility with the current master.
Already done:
- drop support for python2
- switch from PyQt4 to PyQt5

**Comment by thorade** [comment]
Can that branch be merged to the master?

**Comment by thorade** [comment]
closed by PR #6
